docs/index-en.html.bak

```diff
@@ -0,0 +1,4519 @@
+<!DOCTYPE html>

```

Comment:
Why do we have a backup file in this repository?

Reply:
Sorry, I could not get it. Why are you asking that?